qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -472,6 +472,65 @@ def export_to_file(self, filename=None,
                                                   file_type=file_type,
                                                   coordinates=coordinates)
 
+    def export(self, coordinates=CYLINDRICAL, dimensionless_rabi_rate=True):
+        """ Returns a dictionary formatted for plotting using the qctrl-visualizer package.
+
+        Parameters
+        ----------
+        dimensionless_rabi_rate: boolean
+            If True, normalizes the Rabi rate so that its largest absolute value is 1.
+        coordinates: string
+            Indicates whether the Rabi frequency should be plotted in terms of its
+            'cylindrical' or 'cartesian' components.
+
+        Returns
+        -------
+        dict
+            Dictionary with plot data that can be used by the plot_controls
+            method of the qctrl-visualizer package. It has keywords 'Rabi rate'
+            and 'Detuning' for 'cylindrical' coordinates and 'X amplitude', 'Y amplitude',
+            and 'Detuning' for 'cartesian' coordinates.
+
+        Raises
+        ------
+        ArgumentsValueError
+            Raised when an argument is invalid.
+        """
+
+        if coordinates not in [CARTESIAN, CYLINDRICAL]:
+            raise ArgumentsValueError(
+                'Unsupported coordinates provided: ',
+                arguments={'coordinates': coordinates})
+
+        if dimensionless_rabi_rate:
+            normalizer = self.maximum_rabi_rate
+        else:
+            normalizer = 1
+
+        plot_dictionary = {}
+
+        plot_x = self.amplitude_x/normalizer
+        plot_y = self.amplitude_y/normalizer
+        plot_r = self.rabi_rates/normalizer
+        plot_theta = self.azimuthal_angles
+        plot_durations = self.durations
+        plot_detunings = self.detunings
+
+        if coordinates==CARTESIAN:
+            plot_dictionary["X amplitude"] = [{'value': v, 'duration': t}
```

Comment:
Here we are using `X Amplitude` as opposed to `x_amplitude`. However, in the `get_plot_formatted_arrays` it is `rabi_rates` and not `Rabi Rates`.  I would go for one policy everywhere.